test/spec/spec.model.js

```diff
@@ -240,6 +241,22 @@ describe('Model model', function () {
             model.url.should.have.been.calledWithExactly({ url: 'foo' });
         });
 
+        it('can handle a parsed URL object', function () {
+            var url = {
+                protocol: 'http:',
+                port: '1234',
+                hostname: 'proxy-example.com',
+                path: '/'
+            };
+            model.url = sinon.stub().returns(url);
```

Comment:
In a lot of our custom models we're creating a new `url` method that returns the url. I guess that's what I was thinking of when writing this.